client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -36,7 +36,6 @@ function ReaderSubscriptionListItem( {
 	const siteIcon = get( site, 'icon.img' );
 	const feedIcon = get( feed, 'image' );
 	const streamUrl = getStreamUrl( feedId, siteId );
-	const siteUrl = url || getSiteUrl( { feed, site } );
```

Comment:
A new example where this is necessary just sprang up. Feed searching returns back the wrong `urls` for keying into follows. So we are forced to grab it from the feed/site.